src/stm32/usbotg.c

```diff
@@ -382,30 +410,33 @@ OTG_FS_IRQHandler(void)
     }
 }
 
-DECL_CONSTANT_STR("RESERVE_PINS_USB", "PA11,PA12");
-
 // Initialize the usb controller
 void
 usb_init(void)
 {
     // Enable USB clock
+#if CONFIG_MACH_STM32H7
+    if (READ_BIT(PWR->CR3, PWR_CR3_USB33RDY) != (PWR_CR3_USB33RDY) ? 1 : 0) {
```

Comment:
What does this `if` do?  I don't understand the `? 1 : 0` at the end.

-Kevin

Reply:
The end test is not needed, I guess I missed that on cleanup. I had gotten that form the stm32h7 library I was referencing for checking the USB 3.3V to be ready. I will remove the "? 1 : 0" from the if-statement and re-test.

Reply:
I have removed the extra logic from within the if-statement and tested the changes. Thanks for questioning that!